tests/e2e/TestConstants.ts

```diff
@@ -92,7 +92,7 @@ export const TestConstants = {
     /**
      * Delay in milliseconds between checking plugin precence.
      */
-    TS_SELENIUM_PLUGIN_PRECENCE_POLLING: Number(process.env.TS_SELENIUM_PLUGIN_PRECENCE_POLLING) || 2000,
+    TS_SELENIUM_PLUGIN_PRESENCE_POLLING: Number(process.env.TS_SELENIUM_PLUGIN_PRECENCE_POLLING) || 2000,
```

Comment:
Maybe better to use existing ```TS_SELENIUM_DEFAULT_POLLING``` constant and get rid of an extra environment variables?